public/lib/helpers/timeline_helper.js

```diff
@@ -122,6 +122,22 @@ define(function (require) {
       return sortObj;
     };
 
+    /**
+     * Creates an Elasticsearch sort object to sort in chronological order on end field
+     *
+     * @param params group configuraton parameters
+     * @returns Elasticsearch sort object
+     */
+    TimelineHelper.prototype.getSortOnEndFieldObject = function (params) {
+      const sortObj = {};
+      if (params.endFieldSequence) {
+        sortObj[params.endFieldSequence.join('.')] = { order: 'asc' };
```

Comment:
All these things are going to be changed due to discussion with @szydan in this PR ;)